project_task_code/views/project_view.xml

```diff
@@ -26,18 +26,9 @@
         <field name="model">project.task</field>
         <field name="inherit_id" ref="project.view_task_kanban" />
         <field name="arch" type="xml">
-            <xpath
-                expr="//strong[hasclass('o_kanban_record_title')]/t/field[@name='name']"
-                position="before"
-            >
-                <span>[<field name="code" />] </span>
-            </xpath>
-            <xpath
-                expr="//strong[hasclass('o_kanban_record_title')]/s/field[@name='name']"
-                position="before"
-            >
-                <span>[<field name="code" />] </span>
-            </xpath>
+            <field name="name" position="before">
+                <field name="code" />
+            </field>
```

Comment:
This change is weird. I don't think it works.
16.0 has a `t-if= / t-else=`, same as 17.0